src/main.rs

```diff
@@ -185,3 +196,53 @@ fn parse_args() -> Result<LndCfg, ArgsError> {
 
     Ok(LndCfg::new(address, cert_file, macaroon_file))
 }
+
+/// Sets the onion messaging feature bit (described in this PR:
+/// https://github.com/lightning/bolts/pull/759/), to signal that we support
+/// onion messaging. This needs to be done every time we start up, because LND
+/// does not currently persist the custom feature bits that are set via the RPC.
+async fn set_feature_bit(mut client: Client) {
```

Comment:
If we make this `client: & mut Client` then pass in `& mut client` above, I think that we get around some of the move issues / don't need to clone as much.